spec/requests/icons_spec.rb

```diff
@@ -91,4 +91,17 @@
       end
     end
   end
+
+  describe "#override_icon" do
+    let!(:new_icon) { create(:icon, :tenant_id => tenant.id) }
+
+    it "overrides the icon" do
+      post "#{api}/icons/#{new_icon.id}/override", :params => { :portfolio_item_id => portfolio_item.id }, :headers => default_headers
+
+      expect(response).to have_http_status(:ok)
+      expect(json["id"]).to eq new_icon.id.to_s
```

Comment:
Should we have an expectation that the original icon was something other than `new_icon.id`?

Reply:
yep definitely, i added an expectation to check and make sure the new icon is properly set on the portfolio_item.